tox.ini

```diff
@@ -15,18 +15,17 @@ deps =
     .
     readthedocs-sphinx-ext
     pytest
+    sphinxcontrib-httpdomain
     sphinx13: Sphinx < 1.4
     sphinx14: Sphinx < 1.5
     sphinx15: Sphinx < 1.6
     sphinx16: Sphinx < 1.7
     sphinx17: Sphinx < 1.8
     sphinx18: Sphinx < 1.9
+    sphinx20: Sphinx < 2.1
+    sphinx21: Sphinx < 2.2
+    sphinx22: Sphinx < 2.3
+    sphinx23: Sphinx < 2.4
 commands =
-    py.test {posargs}
-
-[testenv:docs]
-changedir = {toxinidir}/docs
-deps =
-    -r{toxinidir}/docs/requirements.txt
-commands =
-    sphinx-build -b html -d {envtmpdir}/doctrees .  {envtmpdir}/html
+    py.test {posargs} tests/
+    sphinx-build -b html -d {envtmpdir}/doctrees docs/ {envtmpdir}/html
```

Comment:
Building the docs with all these versions of Sphinx fails the tests. It looks like this theme has not been able to build its own documentation with Sphinx < 1.5 for some time.

I can get some tests working by changing the first line of `setup()` in `sphinx_rtd_theme/__init__.py` to:
```py
if sphinx.version_info >= (1, 6, 0):
        app.add_html_theme('sphinx_rtd_theme', path.abspath(path.dirname(__file__)))
```

After that change, the sphinx13 and sphinx14 tests fail with
```
sphinx_rtd_theme/.tox/py27-sphinx13/lib/python2.7/site-packages/sphinx/locale/__init__.py", line 195, in _
    return translators['sphinx'].ugettext(message)
KeyError: 'sphinx'
```
 I suggest to revert these lines and separately address compatibility with Sphinx 1.3 and 1.4 (or remove them from the test), so this PR can land.

Reply:
We've discussed removing support for old versions of Sphinx, there likely is not a lot of benefit to supporting these old versions. Even 1.5 is 4 years old at this point.